src/components/Settings/LLMSettings/DefaultLLMSelector.tsx

```diff
@@ -25,16 +40,40 @@ const DefaultLLMSelector: React.FC<DefaultLLMSelectorProps> = ({ llmConfigs, def
     })
   }
 
+  const deleteLLM = async (modelName: string) => {
+    const confirmDelete = window.confirm(`Are you sure you want to delete the model ${modelName}?`)
+    if (!confirmDelete) return
+    try {
+      await window.llm.deleteLLM(modelName)
+      posthog.capture('delete_llm', {
+        modelName,
+      })
+    } catch (error) {
+      console.error(`Failed to delete model: ${modelName}`, error)
```

Comment:
there should be no error catching here. just make the call outside try catch block

Reply:
FIxed!